mango-core/src/main/scala/org/bdgenomics/mango/models/VariantContextMaterialization.scala

```diff
@@ -78,14 +78,16 @@ class VariantContextMaterialization(@transient sc: SparkContext,
    * @return Map of (key, json) for the ReferenceRegion specified
    * N
    */
-  def stringify(data: RDD[(String, GenotypeJson)]): Map[String, String] = {
-
-    val flattened: Map[String, Array[String]] = data
-      .collect
-      .groupBy(_._1).map(r => (r._1, r._2.map(_._2.toString())))
+  def toJson(data: RDD[(String, GenotypeJson)]): Map[String, Array[GenotypeJson]] = {
```

Comment:
Not sure if this function name is appropriate? The scaladoc is also not in sync with the return type.